### PR TITLE
add April 18 meeting notes

### PR DESCRIPTION
**Pull request by garrying**
I've added light formatting to highlight key votes and dates.

### CODE REVIEW

File: meeting_notes/20160418_meeting-notes.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    - If people would like to add in their name for the meeting minutes they were asked to add it to the Meeting Agenda Pad
+- Status Update, current approach been two-pronged:
+    - Vision and establishing a direction for the project
+    - Prototyping hardware
+- Want for an open process, documenting as we go, to be available for people looking up information
+- Think about preliminary outreach to other Mesh organizations (Seattle, NYC, Detroit)
+
+## Vision
+
+- Looking at the problem definition doc: https://docs.google.com/document/d/1XQiqvkuFiTmaVtZcjmwBUfH5cwZdaYYLXHPqWAB1sB0/edit
```
Comment (dcwalk) [inline meeting_notes/20160418_meeting-notes.md]: Can we make these markdown-friendlier urls? e.g.
[ Brainstorming Gdoc ] / ( https://docs.google.com/document/d/1XQiqvkuFiTmaVtZcjmwBUfH5cwZdaYYLXHPqWAB1sB0/edit )

File: meeting_notes/20160418_meeting-notes.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+    - Self-selection bias
+    - How to make avenues for understanding, and approaching infrastructure
+- Q: Are we going for something completely transparent?
+    - Show something that is not legally binding, friendly safe
+    - Accountability: Known risks
+    - How do we build a sustainable community
+    - Ref: Sao Paulo air quality network
+- *High speed access*
+    - Not priority
+    - Ref: http://www.thestar.com/business/tech_news/2016/04/17/what-became-of-torontos-push-for-free-public-wi-fi.html
```
Comment (dcwalk) [inline meeting_notes/20160418_meeting-notes.md]: Same here... [Toronto Star "What became of Toronto’s push for free public Wi-Fi?"] /  (http://www.thestar.com/business/tech_news/2016/04/17/what-became-of-torontos-push-for-free-public-wi-fi.html)

(can just remove the slash!)

File: meeting_notes/20160418_meeting-notes.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+        - Paid for by BIA (in most cases, downtown London)
+        - Possible matching funds
+        - Montreal: Not for profit wireless group, part funded by the city
+        - Involve City Council
+        - Wireless Toronto: Funding is possible (needs more investigating)
+        - **Are people are most interested in free public access (vote)**
+            - Not the main thing; Not the first consideration: Are they mutually exclusive
+        - Free wireless diagram: http://networkcultures.org/unlikeus/wp-content/uploads/sites/2/2013/03/networktypes.png
```
Comment (dcwalk) [inline meeting_notes/20160418_meeting-notes.md]: Here too! 
[Network Types Diagram] / (http://www.rand.org/about/history/baran.html)

File: meeting_notes/20160418_meeting-notes.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+            - BIA could provide an uplink to the Internet
+        - Is this project mutually exclusive? No
+            - The mesh can hook into an established, provided for (BIA), infrastructure
+        - **Vote: Main concern is not access (majority voted yes)**
```
Comment (dcwalk) [inline meeting_notes/20160418_meeting-notes.md]: We all has consensus about everything! Maybe we don't say 'majority,' we just way we voted?

File: meeting_notes/20160418_meeting-notes.md [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+- Access is #1. Everything else are sub-points
+    - _Key issue_: Own the infrastructure
+- _Possible vision statement_: Advancing mesh technology
+- _Possible vision statement_: We're going to build a network that provides access to the internet as well as affords people automony
+- We are going to build an infrastructure that gives users:
+- Transitioned key issue (original doc structure to a vision statement. See doc)
+- We can't have a completely free mesh net (electrical, shared paid infrastructure, solar panels)
+- Black Box, compared to invisible (Black box)
+
+### WE GOTTA: VISION STATEMENT!!
```
Comment (dcwalk) [inline meeting_notes/20160418_meeting-notes.md]: Haha maybe we clean up my enthusiasm a little bit!

File: meeting_notes/20160418_meeting-notes.md [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+        - Role-play (game to teach networking concepts)
+        - Exhibit idea: LEDs light up to show packet travel. Consider families interaction
+        - _Ingrid Burringtons'_ workshop materials as reference
+        - Idea: Water flow diagram to show networking concepts
+        - Idea: Cool app deployment (game, chat)
+        - Idea: Trading zone (barter network, physical network via mesh network)
+        - **Leading proposal writing and submission (April 30): Udit, Ben, Dawn**
+        - **End of May**: Validation of tech (Prototype Developed)
+        - **Mid-June**: Working version will be ready for deployment (Mesh Prototype Validated)
+    - Working sub-groups:
```
Comment (dcwalk) [inline meeting_notes/20160418_meeting-notes.md]: Can we break this out into it's own heading? I think this is super important!!

File: meeting_notes/20160418_meeting-notes.md [hunk 7]
```diff
@@ -0,0 +1,155 @@
...
+        - **Design**: Matt
+        - **Knowledge Management**: Matt, Dawn
+        - **Website**: Dawn, Matt, Garry
+    - Mesh networks 101
+- Website:
+    - Possible Wordpress deployment / _Github_ pages
+    - Deadline for Maker Faire submission (needed for submission)
+- _HOPE_:
+    - Connect with _NYCMesh_
+- **July: schedule a 6-month planning sessions (recap 4 months)**
```
Comment (dcwalk) [inline meeting_notes/20160418_meeting-notes.md]: this as well maybe broken out!

**Comment by dcwalk** [comment]
Hey @garrying looks good! I added in some line notes, typically I through a short recap at the bottom under "# Outcomes" maybe we could throw one or two there?

Also -- I found these notes great! Super detailed, but at points there was almost too much to parse out, especially around the vision statement. Could we trim in a few places just to keep the focus on the key decisions?

Happy to assist!

**Comment by garrying** [comment]
@dcwalk Yeah. It would be great if you could do a pass on trimming if you have the time!

**Comment by garrying** [comment]
@dcwalk Also, I've made updates from your comments. Feel free to make additional edits if we missed anything.

**Comment by dcwalk** [comment]
Look great!

**Pull request merged**